Rakefile

```diff
@@ -17,4 +17,4 @@ task :update_ssl_ca_file do
 end
 
 task checks: %i[rubocop rspec]
-task default: %i[rubocop rspec]
+task default: %i[rspec]
```

Comment:
@unicolet What was the reason for dropping the linter/formatter here? I haven't used Ruby/rubocop, was just curious =)

Reply:
@dawn-minion apparently the sources are so I left it alone

Reply:
Ah ok - no worries then.